app/components/listing/RelativeOpeningTime.jsx

```diff
@@ -2,6 +2,13 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { buildHoursText } from '../../utils';
 
+const parseAsDate = (fourDigitNumber) => {
```

Comment:
Need to adapt this to work with 3 digit numbers for 00-959 but this fix will give us a bunch of different strings including Closed, Closed today, Open, Open 24/7, Open 24 hours today and opens in N minutes and closes in N minutes if N < 30

Reply:
I think this was already implemented in [utils/index.js](https://github.com/ShelterTechSF/askdarcel-web/blob/eb14b43f8ab78def845b417cde714fddcc2822a3/app/utils/index.js#L21). If that implementation is incomplete, then we should replace that one with this, but I think that one handles the 3-character times.

Reply:
Yep I decided not to use the other one because this does more and is a bit more maintainable. Even with the utils function, different places had different implementations.